AUTHORS.md

```diff
@@ -1,45 +1,23 @@
-A full developer contributors list can be found [here](https://github.com/vector-im/element-android/graphs/contributors).
+A full developer contributors list can be found [here](https://github.com/dinsic-pim/tchap-android-v2/graphs/contributors).
```

Comment:
Don't you think better to link to the element-android authors: "https://github.com/vector-im/element-android/blob/master/AUTHORS.md"?

Reply:
Why not. What's happen if we have other contributors on Tchap v2 ?

Reply:
we should put both:
A full element-android developer contributors list can be found [here](https://github.com/vector-im/element-android/graphs/contributors).
A full tchap-android developer contributors list can be found [here](https://github.com/dinsic-pim/tchap-android-v2/graphs/contributors).

Reply:
tchap-android developer contributors list includeselement-android developer contributors list

Reply:
ok - ignore my remark ;)